memory_model/src/guest_address.rs

```diff
@@ -109,13 +109,12 @@ mod tests {
     }
 
     #[test]
-    #[allow(clippy::eq_op)]
     fn cmp() {
-        let a = GuestAddress(0x300);
-        let b = GuestAddress(0x301);
-        assert!(a < b);
-        assert!(b > a);
-        assert!(!(a < a));
+        for i in 1..10 {
```

Comment:
Why is this better than assert!(GuestAddress(0x300), GuestAddress(0x301))?

Reply:
Since this actually tests all possible orderings between the 2 operands of <.